src/MoBi.Presentation/Nodes/TreeNodeFactory.cs

```diff
@@ -126,8 +126,10 @@ public ITreeNode CreateFor(ModuleConfigurationDTO moduleConfiguration)
 
          addModuleBuildingBlocks(moduleConfigurationNode, module);
 
-         createAndAddNodeUnder(moduleConfigurationNode, moduleConfiguration.SelectedMoleculeStartValues);
-         createAndAddNodeUnder(moduleConfigurationNode, moduleConfiguration.SelectedParameterStartValues);
+         if(moduleConfiguration.HasMoleculeStartValues())
```

Comment:
sometimes the selected start values are null, other times, they are static Null objects

Reply:
It tests for the special null object